src/containers/Articles/Article/CreateArticle.jsx

```diff
@@ -93,17 +102,46 @@ export class CreateArticle extends Component {
             body,
             description
         } = this.state.articleData;
+        const { tags } = this.state;
         const dataToFormData = new FormData();
         if (image !== null) {
-            dataToFormData.append("image", image, image.name)
-        };
+            dataToFormData.append("image", image, image.name);
+        }
+        if (tags !== undefined || tags.length !== 0) {
+            tags.map(tag => {
+                dataToFormData.append("tag_list", tag.text);
+                return tag.text;
+            });
+        }
         dataToFormData.append("title", title);
         dataToFormData.append("body", body);
         dataToFormData.append("description", description);
 
         this.props.onCreateArticle(dataToFormData);
     }
 
+    handleDelete = (i) => {
+        const { tags } = this.state;
+        this.setState({
+            tags: tags.filter((tag, index) => index !== i),
+        });
+    }
```

Comment:
Good work @codjoero but shouldn't these methods have semi-colons at the end `handleDelete`, `handleAddition` and `handleDrag` as well as I don't think there's need for parenthesis on `handleDelete` and `handleAddition` as they take in just one argument each.